### PR TITLE
MFA related documentation changes.

### PR DESCRIPTION
**Pull request by alesstimec**

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -53,33 +57,52 @@ with `go install gopkg.in/macaroon-bakery.v2/cmd/bakery-keygen` to generate
...
 This is a list of the configured identity providers with their
 configuration. See below for the supported identity providers. If this
 is not configured then a default set of providers will be used
 containing the Ubuntu SSO and Agent identity providers.
 
 ### api-macaroon-timeout
+
 This is the maximum time a login to the /v1 API will remain logged
-in for. As candid uses itself as it's authentication provider,
+in for. As candid uses itself as it's authentication provider,
```
Comment (mhilton) [inline docs/configuration.md]: This is almost certainly my error, but it should be "its", not "it's".

Comment (mhilton) [reply docs/configuration.md]: also you seem to have added an unnecessary space.

File: docs/configuration.md [hunk 2]
```diff
@@ -53,33 +57,52 @@ with `go install gopkg.in/macaroon-bakery.v2/cmd/bakery-keygen` to generate
...
 containing the Ubuntu SSO and Agent identity providers.
 
 ### api-macaroon-timeout
+
 This is the maximum time a login to the /v1 API will remain logged
-in for. As candid uses itself as it's authentication provider,
+in for. As candid uses itself as it's authentication provider, 
 for all practical purpose the login time will be the minimum of
-`api-macaroon-timeout` and `discharge-macaroon-timeout`. The default
+`api-macaroon-timeout` and `discharge-macaroon-timeout` . The default
```
Comment (mhilton) [inline docs/configuration.md:77]: Why did you add a space here?

File: docs/configuration.md [hunk 3]
```diff
@@ -53,33 +57,52 @@ with `go install gopkg.in/macaroon-bakery.v2/cmd/bakery-keygen` to generate
...
 the target service may also have it's own maximum time.
 
 ### discharge-token-timeout
+
 This is the maximum time that the discharge token issued to the client
 can be used to discharge tokens without requiring re-authentication.
 
+### mfa-rp-display-name
+
+This is the name of the cadid as a relying party for the multi-factor
```
Comment (mhilton) [inline docs/configuration.md]: s/cadid/candid/

File: docs/configuration.md [hunk 4]
```diff
@@ -103,10 +126,10 @@ all data ephemerally in RAM.
 
 This uses MongoDB for the backend. It has two parameters:
 
-`address` (required) is the address of the mongoDB server to connect to,
+`address` (required) is the address of the mongoDB server to connect to,
```
Comment (mhilton) [inline docs/configuration.md:129]: more confusing spaces?

**Review by mhilton — APPROVED** [review]
LGTM, with a few thoughts.

File: docs/configuration.md (hunk 1 above)
Comment (alesstimec) [reply docs/configuration.md]: fixed the first part.. as for spaces, vs code internal formatter added them..

File: docs/configuration.md (hunk 3 above)
Comment (alesstimec) [reply docs/configuration.md]: fixed
